server/src/modules/repo/users-repo.service.ts

```diff
@@ -297,6 +297,15 @@ export class UsersRepoService {
         return [mapLibraryEntries, count];
     }
 
+	async addMapLibraryEntry(userID: number, mapID: number) {
```

Comment:
Lets get naming consistent with rest of repo service and use `createMapLibraryEntry`.